datacollection/openapi/v3/schema/adsb.yaml

```diff
@@ -61,35 +56,307 @@ components:
+          secondaryAltitudeMM:
+            type: integer
+            description: Geometric altitude or barometric pressure altitude in millimeters
+          tisBSiteId:
+            type: integer
+            description: The tisBSiteId is unit-less and is from the a transmitted TISb UAT message signifies which uplink tower transmitted the TISb frame
+          transmitMSO:
+            type: integer
+            enum: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20,21,22,23,24,25,26,27,28,29,30,31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50,51,52,53,54,55,56,57,58,59,60,61,62,63]
+            description: the transmitMSO is the 6bit field from the transmitted UAT message which should signify which MSO the message was transmitted in.  MSO's can range from 0 to 3951 but only transmit the 6 LSB's of the actual MSO if transmitted.
+          addressQualifier:
+            type: integer
+            enum: [0,1,2,3,4,5,6,7]
+            description: Defines the type of target that delivered the data
+              0 = ADS-B target with ICAO 24-bit
+              1 = Reserved for National use
+              2 = TIS-B target with ICAO 24-bit address
+              3 = TIS-B target with track file identifier
+              4 = Surface Vehicle
+              5 = Fixed ADS-B Beacon
+              6-7 = (Reserved)
+          uatMopsVersion:
+            type: integer
+            enum: [0,1]
+            description: 1 = DO-282A, 2 = DO-282B
+          callSignID:
+            type: integer
+            enum: [0,1]
+            description: 0 = Fightplan, 1 = CallSign
+
+  #    RID:
+  #      type: object
+  #      properties:
+          uvin:
+            type: string
+            format: uuid
+          toLatDD:
+            type: number
+            description: Takeoff latitude expressed as decimal degrees
+          toLonDD:
+            type: number
+            description: Takeoff longitudeexpressed as decimal degrees
+          missionType:
+            type: integer
+            enum: [0,1,2,3,4,5,6,7,8,9,10,11]
+            description: Numeric representation of the type of mission being flown
+              0 = recreational
+              1 = photography/videography
+              2 = media
+              3  = point/structure inspection
+              4 = agriculture
+              5 = mapping
+              6 = surveillance
+              7 = law enforcement / first responder
+              8 = search and rescue
+              9 = delivery
+              10= urgent delivery
+              11 =other
+          phoneNumber:
+            type: integer
+            description: Phone number attached to the mission.
+              todo sic. Cannot possibly be right. long or string?
+          fltplanFiled:
+            type: integer
+            enum: [0,1]
+            description: NOTAM/UOA flight plan filed flag
+              todo, not checked, no example
 
+# timestamp stolen from .. todo: should use commons
```

Comment:
timestamp is hardest part of codegen.  What should this be

Reply:
It's a date/time string formatted according to ISO8601, as referenced by https://tools.ietf.org/html/rfc3339#section-5.6, as referenced by https://swagger.io/docs/specification/data-models/data-types/

I don't think I know enough about the intended downstream users to specify a particular approach in codegen. I'm trying to provide as much information for JSON validation to the endpoint as I can.

I can remove the comment, and I'm not convinced that we should tie this to our published commons. uAvionics could change it (although I doubt they will).